workflows/sample/tasks/hifiasm_trio.wdl

```diff
@@ -207,30 +177,6 @@ workflow hifiasm_trio {
       pb_conda_image = pb_conda_image
   }
 
-  call yak_trioeval as yak_trioeval_p_ctg  {
-    input:
-      fasta_gz = bgzip_fasta_p_ctg.fasta_gz,
-      parent1_yak = parent1_yak,
-      parent2_yak = parent2_yak,
-      pb_conda_image = pb_conda_image
-  }
-
-  call yak_trioeval as yak_trioeval_p_utg  {
-    input:
-      fasta_gz = bgzip_fasta_p_utg.fasta_gz,
-      parent1_yak = parent1_yak,
-      parent2_yak = parent2_yak,
-      pb_conda_image = pb_conda_image
-  }
-
-  call yak_trioeval as yak_trioeval_r_utg  {
-    input:
-      fasta_gz = bgzip_fasta_r_utg.fasta_gz,
-      parent1_yak = parent1_yak,
-      parent2_yak = parent2_yak,
-      pb_conda_image = pb_conda_image
-  }
-
```

Comment:
`yak trioeval` is computationally expensive, and only really makes sense to run on the finished assembly hap1 and hap2.

Reply:
I'm not even sure we want to run trioeval at all.  @byoo, are you using the trioeval results?

Reply:
Will change to conditional run